### ISSUE TITLE
Improve backend plugins logs

### ISSUE THREAD

**Issue by alexanderzobnin** [grafana/grafana#17111]
Right now logs from plugins look unstructured - some fields are duplicated with Grafana logs:
```
# Logs from grafana
DBUG[05-16|12:29:06] starting plugin                          logger=plugins plugin-id=grafana-simple-json-backend-datasource path=/home/alex/development/go/src/github.com/grafana/simple-json-backend-datasource/dist/simple-json-plugin_linux_amd64 args=[/home/alex/development/go/src/github.com/grafana/simple-json-backend-datasource/dist/simple-json-plugin_linux_amd64]
DBUG[05-16|12:29:06] plugin started                           logger=plugins plugin-id=grafana-simple-json-backend-datasource path=/home/alex/development/go/src/github.com/grafana/simple-json-backend-datasource/dist/simple-json-plugin_linux_amd64 pid=18802
DBUG[05-16|12:29:06] waiting for RPC address                  logger=plugins plugin-id=grafana-simple-json-backend-datasource path=/home/alex/development/go/src/github.com/grafana/simple-json-backend-datasource/dist/simple-json-plugin_linux_amd64
# Logs from plugin
DBUG[05-16|12:29:06] 2019-05-16T12:29:06.492+0300 [DEBUG] simple-json-backend-datasource: Running Simple JSON backend datasource logger=plugins plugin-id=grafana-simple-json-backend-datasource
DBUG[05-16|12:29:06] 2019-05-16T12:29:06.492+0300 [DEBUG] simple-json-backend-datasource: plugin address: network=unix address=/tmp/plugin597538605 logger=plugins plugin-id=grafana-simple-json-backend-datasource
```

Problems:
- Time and log level are duplicated
- Developer can't set log level in plugin (it will be `DBUG[05-16|12:29:06] 2019-05-16T12:29:06.492+0300 [INFO] simple-json-backend-datasource`, but should be `INFO[05-16|12:29:06] 2019-05-16T12:29:06.492+0300 [INFO] simple-json-backend-datasource`)

Hashicorp [docs](https://github.com/hashicorp/go-plugin#features) says:
>If the host system uses hclog then the log data will be structured. If the plugin also uses hclog, logs from the plugin will be sent to the host hclog and be structured.

So I think it would be nice to get structured plugin logs inside grafana and format it as normal logs. I'm not sure how to achieve that, but hopefully, it's possible.

**Comment by torkelo** [comment]
please label new issues backend or frontend, plus a feature area

**Comment by marefr** [comment]
Should be fixed by #20514 from Grafana point of view. Plugin still need to use hclog with json format to get proper output in Grafana log.
